app/js/services/resultatService.js

```diff
@@ -2,20 +2,25 @@
 
 angular.module('ddsApp').service('ResultatService', function($http, $modal, droitsDescription) {
 
+    var DROITS_POTENTIELS = droitsDescription.prestationsNationales.concat(
+        _.flatten(_.pluck(droitsDescription.partenairesLocaux, 'prestations'))
```

Comment:
You mean `Array.prototype.concat.apply` I guess :rabbit: 
I understand the idea, but it is not super transparent for me. Is `Array.prototype.concat.apply` the same than `[].concat.apply` ?